sklearn/metrics/tests/test_common.py

```diff
@@ -472,8 +486,11 @@ def test_sample_order_invariance_multilabel_and_multioutput():
 @ignore_warnings
 def test_format_invariance_with_1d_vectors():
     random_state = check_random_state(0)
-    y1 = random_state.randint(0, 2, size=(20, ))
-    y2 = random_state.randint(0, 2, size=(20, ))
+    # Changing from randint(0, 2) to randint(1, 3) to generalize
```

Comment:
Don't say you're changing: it'll be this way forever :)

Reply:
yeah :), I'll rectify